tests/testsuite/git.rs

```diff
@@ -283,7 +283,7 @@ fn cargo_compile_git_dep_pull_request() {
         .with_stderr_data(str![[r#"
 [UPDATING] git repository `[ROOTURL]/dep1`
 [LOCKING] 1 package to latest compatible version
-[COMPILING] dep1 v0.5.0 ([ROOTURL]/dep1?rev=refs/pull/330/head#[..])
+[COMPILING] dep1 v0.5.0 ([ROOTURL]/dep1?rev=refs%2Fpull%2F330%2Fhead#[..])
```

Comment:
Oof, slashes in query strings need percent encoding? Thats too bad.

Reply:
This makes me wonder if we did the wrong encoding…

Reply:
Huh, the wikipedia article i read didn't mention slashes being allowed.